tests/guest/GuestTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+namespace Tests\guest;
+
+use Tests\TestCase;
+
+/*
+ * guest/:get guest/$id:get
+ */
+
+use App\Models\Guest;
+use App\Models\Term;
+use App\Models\User;
+use Illuminate\Support\Str;
+
+class GuestTest extends TestCase {
+    public function testGetAll() {
+        $term_count = 3;
+        $guest_count = 3;
+        $user = User::factory()->permission('executive')->create();
+
+        for ($i = 0; $i < $term_count; $i++) {
+            Guest::factory()->count($term_count)->for(Term::factory())->count($guest_count)->create();
+        }
+
+        $this->actingAs($user)->get('/guests');
+        $this->assertResponseOk();
+        $this->assertJson($this->response->getContent());
+        $res = json_decode($this->response->getContent());
+        $this->assertCount($term_count * $guest_count, $res);
+    }
+
+    public function testShow() {
+        $user = User::factory()->permission('executive')->create();
+        $guest = Guest::factory()->create();
+
+        $this->actingAs($user)->get("/guests/$guest->id");
```

Comment:
```suggestion
        $this->actingAs($user)->get("/guests/{$guest->id}");
```
動くのは分かるけど明示的に括っておきたい

Reply:
無言無修正close何
(slackに書いてるならそれはここに書けとなる)

Reply:
よく見たら下に差分がある…？ なにこれ？

Reply:
あ 1行単位でaddしたときに削除行をaddするのを忘れてました(ごめんなさい)